core/txpool/validation.go

```diff
@@ -395,6 +409,14 @@ func ValidateTransactionWithState(tx *types.Transaction, signer types.Signer, op
 				return ErrAddressBlacklisted
 			}
 		}
+
+		// Verify no authorizations will invalidate existing transactions known to
+		// the pool.
+		if opts.KnownConflicts != nil {
```

Comment:
this block of code is placed in a wrong place, fixed in the 7009548935d07dd467caaf25c251b51f872a739e